docs/data/date-pickers/date-range-calendar/CustomDateRangePickerDay.tsx

```diff
@@ -40,10 +41,12 @@ const DateRangePickerDay = styled(MuiDateRangePickerDay)(
 export default function CustomDateRangePickerDay() {
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
-      <DateRangeCalendar
-        defaultValue={[dayjs('2022-04-07'), dayjs('2022-04-10')]}
-        slots={{ day: DateRangePickerDay }}
-      />
+      <Box sx={{ overflow: 'auto' }}>
```

Comment:
Maybe just wrap it in `DemoContainer` like in other demos? 🤔

Reply:
It looks like `<DemoContainer>` makes sense when we need to add the labels on top of the components, otherwise, it looks like an over-abstraction, I mean this `components` prop looks painful to use and understand. I think that Box and Stack are simpler.

For example in http://0.0.0.0:3001/x/react-date-pickers/base-concepts/#controlled-value, it looks like only overhead for developer that want to understand what the demo does.

Reply:
Yes, it indeed is a somewhat bigger overhead, but we decided to use it in order to stay sane when trying to manage the layout of various pickers and fields and their respective styles for various viewports.
In order to correctly apply styles to the various demos, we'd have to use a lot of code duplication.
That's why we decided to go for `DemoContainer` and hide it in previews. 🤔